crates/anvil-polkadot/src/substrate_node/chain_spec.rs

```diff
@@ -110,16 +117,48 @@ fn props() -> Properties {
 
 pub fn development_chain_spec(
     genesis_config: GenesisConfig,
+    signers: &[Keypair],
+    genesis_balance: U256,
 ) -> Result<DevelopmentChainSpec, String> {
+    let genesis_balance = SubstrateU256::from(genesis_balance).inner().as_u128();
+    let mut balance_map: BTreeMap<AccountId32, u128> = signers
+        .into_iter()
+        .map(|keypair| (Account::from(keypair.clone()).substrate_account(), genesis_balance))
+        .collect();
+    if let Some(alloc) = &genesis_config.alloc {
+        balance_map.extend(alloc.iter().map(|(eth_addr, gen_account)| {
+            let mut substrate_account = AccountId32::new([0xEE; 32]);
+            <AccountId32 as AsMut<[u8; 32]>>::as_mut(&mut substrate_account)[..20]
+                .copy_from_slice(eth_addr.as_slice());
+            let balance = gen_account.balance.try_into().unwrap_or(genesis_balance);
```

Comment:
this balance is probably not right, since substrate balances use fewer decimals to express the balance (which is why we have the dust stored in pallet-revive)

Reply:
yeah, that would be a good reason to merge #357 first